moveit_setup_assistant/src/widgets/configuration_files_widget.cpp

```diff
@@ -1140,6 +1153,9 @@ bool ConfigurationFilesWidget::addTemplateString(const std::string& key, const s
 // ******************************************************************************************
 bool ConfigurationFilesWidget::copyTemplate(const std::string& template_path, const std::string& output_path)
 {
+  // clear template strings in order to update them
+  template_strings_.clear();
+
   // Check if template strings have been loaded yet
   if (template_strings_.empty())
```

Comment:
It does not make sense to clear the list and then condition on whether it is empty.

Reply:
True, I'll remove the conditional (and caching) if that's ok?

Reply:
Why did you disable the caching in the first place?
I don't know the speedup is really significant, but I don't see why it should break your new feature?

Reply:
Well, this cache has no way of being invalidated by user-input so basically going back and changing things after running export once would not lead to updates in the exported data. I and probably also previous contributors thought this would only set static content (ie based on the urdf file not on user input) but looking at it this has not been true anyway.

I found an inbetween solution that I'll commit in a second. Re-exporting is still quite buggy visually but at least should not loose data.